ts/features/itwallet/common/hooks/useItwDismissalDialog.tsx

```diff
@@ -3,19 +3,28 @@ import { useHardwareBackButton } from "../../../../hooks/useHardwareBackButton";
 import I18n from "../../../../i18n";
 import { useIONavigation } from "../../../../navigation/params/AppParamsList";
 
+type ItwDismissalDialogProps = {
+  handleDismiss?: () => void;
+  customBodyMessage?: string;
+};
+
 /**
  * Allows to show a dismissal dialog in which the user must confirm the desire to close the current flow.
  * This hook also handles the hardware back button to show the dialog when the user presses the back button.
  * @param handleDismiss - An optionalfunction that will be called when the user confirms the dismissal.
+ * @param customBodyMessage - An optional custom message to be shown in the dialog body.
  * @returns a function that can be used to show the dialog
  */
-export const useItwDismissalDialog = (handleDismiss?: () => void) => {
+export const useItwDismissalDialog = (props?: ItwDismissalDialogProps) => {
```

Comment:
We don't need to pass an object, we can use multiple parameters and make it retrocompatible. What do you think?
```suggestion
export const useItwDismissalDialog = (handleDismiss?: () => void;
  customBodyMessage?: string;) => {
```

Reply:
Uhm, I thought `handleDismiss` was always passed as argument. But you are right. If it can be `undefined` it's better to pass an object instead.